twitter/twitter.php

```diff
@@ -1940,7 +1940,9 @@ function twitter_fetchhometimeline(App $a, $uid)
 			if (empty($postarray['thr-parent'])) {
 				$contact = DBA::selectFirst('contact', [], ['id' => $postarray['contact-id'], 'self' => false]);
 				if (DBA::isResult($contact)) {
-					$notify = Item::isRemoteSelf($contact, $postarray);
+					if (Item::isRemoteSelf($contact, $postarray)) {
+						$notify = PRIORITY_MEDIUM;
+					}
```

Comment:
Please combine this condition with the "parent" one:
```php
				if (DBA::isResult($contact) && Item::isRemoteSelf($contact, $postarray)) {
```

Reply:
Done